### PR TITLE
Watteco devices codec with decode_uplink

### PR DESCRIPTION
**Pull request by Logia556**
contains the WATTECO devices. Thank you for reviewing this pull request and give feedbacks

### CODE REVIEW

File: vendors/watteco/drivers/decode.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            try {
+                let decoded = batch.normalisation_batch(batchInput)
+                return {
+                    data: decoded,
+                    warnings: [""],
+                }
+            } catch (error) {
+                return {
+                    error: error.message,
+                    warnings: [""],
```
Comment (mostafaibr) [inline vendors/watteco/drivers/decode.js:29]:
```suggestion
                    warnings: [],
```

File: vendors/watteco/drivers/decode.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                batch1: batch_parameters[0],
+                batch2: batch_parameters[1],
+                payload: payload,
+                date: date,
+            }
+            try {
+                let decoded = batch.normalisation_batch(batchInput)
+                return {
+                    data: decoded,
+                    warnings: [""],
```
Comment (mostafaibr) [inline vendors/watteco/drivers/decode.js:24]:
```suggestion
                    warnings: [],
```

File: vendors/watteco/drivers/standard.js [hunk 3]
```diff
@@ -0,0 +1,3379 @@
...
+        else
+        {
+            decoded.batch = {};
+            decoded.batch.report = "batch";
+        }
+    }
+    return decoded;
+}
+function normalisation_standard(input, endpoint_parameters){
+    let warning = [""];
```
Comment (mostafaibr) [inline vendors/watteco/drivers/standard.js:3239]:
```suggestion
    let warning = [];
```

**Review by mostafaibr — CHANGES_REQUESTED** [review]
Hello @Logia556, 

Thank you for your contribution. 

I have added comments on decode.js and standard.js about `warnings` with code suggestions.
In addition, added a comment on the examples.json file of Clos'o driver about the same issue, this comment applies to all other drivers. 

Can you please fix that in order to be able to merge your PR and integrate the devices into the next release as soon as possible?

The PR has conflicts with the main branch, please consider resolving them. 

Thank you.

**Comment by mostafaibr** [comment]
Another small remark, in the driver there is a lot of console.log statements, could you remove that also?

**Comment by Logia556** [comment]
Empty strings in warning, and console.log() removed.
Conflicts resolved.

**Review by Logia556 — COMMENTED** [review]
empty strings removed in warnings

**Review by mostafaibr — APPROVED** [review]

**Pull request merged**